app/controllers/clients_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ClientsController < ApplicationController
+  load_and_authorize_resource
```

Comment:
@amyshan to authenticate just add this line to the right controller. idk how it works exactly its magic 🎉